arangod/Cluster/ClusterInfo.h

```diff
@@ -807,6 +842,21 @@ class ClusterInfo final  {
 
   application_features::ApplicationServer& server() const;
 
+  //////////////////////////////////////////////////////////////////////////////
+  /// @brief (re-)load the information about our plan
+  /// Usually one does not have to call this directly.
+  //////////////////////////////////////////////////////////////////////////////
+
+  void loadPlan();
```

Comment:
Can we actually make `loadPlan` and `loadCurrent` `private`? As far as I see they are still public here.

Reply:
I tried this and a lot of unit tests still do explicit `loadPlan` and `loadCurrent`. Should we adjust this?